addons/sale_loyalty/tests/test_loyalty.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from odoo import Command
```

Comment:
```suggestion
from odoo.fields import Command
```
Not mandatory, but I personally prefer importing it from fields (where it's defined)